geotrek/tourism/parsers.py

```diff
@@ -1166,21 +1167,24 @@ def link_informationdesks(self, parser, datas, match_id_uuid, json_uuid_key):
         model_imported = parser.model
         field = "information_desks"
         for row in datas['results']:
-            infodesks_to_set = [match_id_uuid.get(val) for val in row[field]
-                                if match_id_uuid.get(val)]
-            # object_result_field is the objects found for each field in initial_fields
-            # example every information desks for one trek
-            current_infodesks = getattr(model_imported.objects.get(
-                **{json_uuid_key: row[json_uuid_key]}),
-                field)
-            infodesks_to_remove = current_infodesks.exclude(
-                id__in=[object_result.pk for object_result in infodesks_to_set])
-            if infodesks_to_remove:
-                current_infodesks.remove(
-                    *infodesks_to_remove
-                )
-            getattr(model_imported.objects.get(**{json_uuid_key: row[json_uuid_key]}), field).add(
-                *infodesks_to_set)
+            try:
+                trek = model_imported.objects.get(**{json_uuid_key: row[json_uuid_key]})
+            except Trek.DoesNotExist:
+                self.add_warning(_("Cannot link information desk to trek: could not find trek with UUID {uuid}")
```

Comment:
don't use format in translated string. Use specific notation : 

_("Today is %(month)s %(day)s.") % {"month": m, "day": d}

https://docs.djangoproject.com/fr/5.1/topics/i18n/translation/#standard-translation